BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01822-01
+2.0.0-prerelease-01901-01
```

Comment:
those capabilities are part of the test runner package, not buildtools. That means that you don't really need to update this version. It's ok if you want to do this anyway on master, but we will definitely not want to take this to release, so I would just advice to remove it.